src/Speckle.Sdk/Serialisation/V2/ServerObjectManager.cs

```diff
@@ -13,6 +13,12 @@
 
 namespace Speckle.Sdk.Serialisation.V2;
 
+public class ServerObjectManagerOptions(TimeSpan? timeout = null, string? boundary = null)
+{
+  public TimeSpan Timeout => timeout ?? TimeSpan.FromSeconds(120);
+  public string Boundary => boundary ?? Guid.NewGuid().ToString();
```

Comment:
What is this boundary for?
and is it ok that `.Boundary` will return a new GUID each time? instead of `??= Guid.NewGuid().ToString()`?

Reply:
actually, it won't...it assigns just one value after computing it lazily.

Reply:
In cases where `boundary` is null, are you saying that `boundary` will be assigned with the new guid?
Because the null-coalescing operator (`??`) doesn't work that way... the null-assignment operator (`??=`) will  assign...

Reply:
You could also use backed properties to achieve the same affect. But in its current form... This is the behaviour
![image](https://github.com/user-attachments/assets/6758b708-e624-459f-8a02-1a282c628727)

Reply:
I was misunderstanding how the => worked for properties.  I see what you mean though and you're right so I changed it.